.env.example

```
@@ -0,0 +1,4 @@
+# VITE_* 값은 브라우저 번들에 공개된다. 인증 비밀값을 절대 넣지 않는다.
+VITE_PUBLIC_BASE_PATH=/pull-it/
+VITE_API_BASE_URL=/pull-it
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_API_BASE_URL key should go before the VITE_PUBLIC_BASE_PATH key

(UnorderedKey)